### PR TITLE
Add style for nav

### PR DESCRIPTION
**Pull request by Nezumi-2711**

### CODE REVIEW

File: practice/practice-02/src/styles/components/_button.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.btn {
+  align-self: center;
+  display: none;
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/components/_button.scss]: Button is display none? I don't think that correct. It only happen for your Download button, not all button

File: practice/practice-02/src/styles/components/_menu-button.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
+.icon-menu {
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/components/_menu-button.scss]: Menu-button is not a good name

File: practice/practice-02/src/styles/bases/reset.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
+$background-color_1: transparent;
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/bases/reset.scss]: Reset file is not correct. Pls refer: https://meyerweb.com/eric/tools/css/reset/

File: practice/practice-02/src/styles/main.scss [hunk 4]
```diff
@@ -1 +1,20 @@
-@import "./base/reset.scss";
+// Abstracts
+@import "./abstracts/mixins";
+@import "./abstracts/variables";
+
+// Bases
+@import "./bases/typography";
+@import "./bases/reset.scss";
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/main.scss]: File name is not correct

File: practice/practice-02/src/styles/main.scss [hunk 5]
```diff
@@ -1 +1,20 @@
...
+// Components
+@import "./components/button";
+@import "./components/menu-button";
+@import "./components/logo";
+
+// Layouts
+@import "./layouts/navbar";
+@import "./layouts/header";
+
+* {
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/main.scss]: Should not be here

File: practice/practice-02/src/styles/components/_button.scss (hunk 1 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/components/_button.scss]: Fixed.

File: practice/practice-02/src/styles/components/_menu-button.scss (hunk 2 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/components/_menu-button.scss]: Changed.

File: practice/practice-02/src/styles/bases/reset.scss (hunk 3 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/bases/reset.scss]: Fixed.

File: practice/practice-02/src/styles/main.scss (hunk 4 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/main.scss]: Fixed.

File: practice/practice-02/src/styles/main.scss (hunk 5 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/main.scss]: Fixed.

File: practice/practice-02/src/styles/components/_button.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  background-color: $primary-color;
+  border-radius: 100px;
+  font-size: $fs-md;
+  color: $white;
+  text-decoration: none;
+  white-space: nowrap;
+  transition: 0.2s;
+}
+
+.nav-download-btn {
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/components/_button.scss]: nav-download-btn should not be here

File: practice/practice-02/src/styles/components/_toggle-menu.scss [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  transform: rotate(-45deg);
+}
+
+.toggle-menu:checked ~ .icon-menu:after {
+  top: 0;
+  transform: rotate(45deg);
+}
+
+.toggle-menu:checked ~ .nav-list {
+  right: 0px;
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/components/_toggle-menu.scss]: right: 0px? Pls compare with code in line 43

File: practice/practice-02/src/styles/layouts/_navbar.scss [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+  height: auto;
+  margin-top: 105px;
+  margin-right: 10px;
+  margin-left: 10px;
+  z-index: 2;
+  background-color: $white;
+}
+
+.nav-item {
+  font-size: 14px;
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/layouts/_navbar.scss]: Use variable if you can

File: practice/practice-02/src/styles/layouts/_navbar.scss [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    display: flex;
+    justify-content: space-between;
+    border: none;
+    margin: auto;
+    max-width: 600px;
+  }
+
+  .nav-item {
+    border-bottom: none;
+    padding: 0px;
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/layouts/_navbar.scss]: As comment for top, bottom with 0 value, pls correct

File: practice/practice-02/src/styles/main.scss (hunk 4 above)
Comment (phuocduong-agilityio) [reply practice/practice-02/src/styles/main.scss]: Why we need scss extension here?

File: practice/practice-02/src/styles/components/_button.scss (hunk 6 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/components/_button.scss]: Fixed.

File: practice/practice-02/src/styles/components/_toggle-menu.scss (hunk 7 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/components/_toggle-menu.scss]: Fixed.

File: practice/practice-02/src/styles/layouts/_navbar.scss (hunk 8 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/layouts/_navbar.scss]: Fixed.

File: practice/practice-02/src/styles/layouts/_navbar.scss (hunk 9 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/layouts/_navbar.scss]: Fixed.

File: practice/practice-02/src/styles/main.scss (hunk 4 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/main.scss]: Sorry for my carelessness, I will fix them.

File: practice/practice-02/src/styles/bases/_reset.scss [hunk 10]
```diff
@@ -0,0 +1,128 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/bases/_reset.scss:1]: Pls format this file

File: practice/practice-02/src/styles/layouts/_header.scss [hunk 11]
```diff
@@ -0,0 +1,27 @@
+.header {
+  position: relative;
+  display: flex;
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/styles/layouts/_header.scss]: I saw you use alot of flex style, could you split it for common uses?

File: practice/practice-02/src/styles/bases/_reset.scss (hunk 10 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/bases/_reset.scss:1]: Fixed.

File: practice/practice-02/src/styles/layouts/_header.scss (hunk 11 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/styles/layouts/_header.scss]: Fixed.

**Pull request merged**